uint8 ProcAlgorithm
string BrickType
---
bit_vision/Task2_vision_msg VisionData